tree/dataframe/src/RDataFrame.cxx

```diff
@@ -1390,6 +1390,10 @@ RDataFrame::RDataFrame(std::unique_ptr<ROOT::RDF::RDataSource> ds, const ColumnN
 {
 }
 
+RDataFrame::RDataFrame(const ROOT::RDF::RDatasetSpec &spec)
+   : RInterface(std::make_shared<RDFDetail::RLoopManager>(spec))
```

Comment:
```suggestion
RDataFrame::RDataFrame(ROOT::RDF::RDatasetSpec spec)
   : RInterface(std::make_shared<RDFDetail::RLoopManager>(std::move(spec)))
```

Reply:
That makes sense.

So to be sure: what we want is: the RDF will take a hard copy of the spec, that is cool.

But then I realize, we don't need a hard copy of the spec for the RLoopManager, because it can take a reference from the RDF. Right?

Reply:
Maybe it can take a `RDatasetSpec &&` through the moved spec coming from the RDataFrame constructor yes